crates/model/src/solver_competition_v2.rs

```diff
@@ -0,0 +1,62 @@
+use {
+    crate::{auction::AuctionId, order::OrderUid},
+    number::serialization::HexOrDecimalU256,
+    primitive_types::{H160, H256, U256},
+    serde::{Deserialize, Serialize},
+    serde_with::serde_as,
+    std::collections::BTreeMap,
+};
+
+/// Stored directly in the database and turned into SolverCompetitionAPI for the
+/// `/solver_competition` endpoint.
+#[derive(Clone, Debug, Default, Deserialize, Serialize, PartialEq)]
+#[serde_as]
+#[serde(rename_all = "camelCase")]
+pub struct Response {
+    pub auction_id: AuctionId,
+    pub auction_start_block: i64,
+    pub transaction_hashes: Vec<H256>,
```

Comment:
This field now feels a bit redundant, as we are now including the `tx_hash` field on the `Solution` definition